.eslintrc

```diff
@@ -23,6 +24,8 @@
     "jsx-quotes"           : [2, "prefer-single"],
     "max-len"              : [2, 120, 2],
     "object-curly-spacing" : [2, "always"],
-    "comma-dangle"         : "off"
+    "comma-dangle"         : "off",
```

Comment:
This was the default setting in react-redux-starter-kit ([https://github.com/davezuko/react-redux-starter-kit/blob/master/.eslintrc](url)). I appended the comma and next 2 rules.
Should I change to the one you provided?